test.sh

```diff
@@ -103,6 +103,18 @@ expect_equal_files() {
     fi
 }
 
+expect_different_files() {
+    if [ ! -e "$1" ]; then
+        test_failed "compare_files: $1 missing"
```

Comment:
Please don't use tabs in shell scripts.

Reply:
There were no tabs in it. Perhaps from the first commit? I've squashed everything and rebased it on master.

Reply:
(There were lots of space→tab indent changes in the first commit which then were changed back in a subsequent commit.)